lib/SingletonApiResource.php

```diff
@@ -22,7 +22,7 @@ protected static function _singletonRetrieve($options = null)
      */
     public static function classUrl()
     {
-        $base = static::className();
+        $base = static::OBJECT_NAME;
```

Comment:
OOC what is that file about? I don't remember seeing it before

Reply:
The only singleton resource is `Balance`. I guess that we expected this to be a more common pattern in the API than it ended up being.